.vs/homework/task11_library/TestsLibrary.cs

```diff
@@ -0,0 +1,47 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using task11_library;
+
+namespace task11_library.Tests
+{
+    public class ReaderUnitTests
+    {
+
+        public void ConstructorTest()
```

Comment:
Тестовый метод должен быть помечен соответствующим атрибутом
```suggestion
        [Test]
        public void ConstructorTest()
```